### ISSUE TITLE
Odd backward navigation after searching for ⲱⲥ

### ISSUE THREAD

**Issue by m-linssen** [KELLIA/dictionary#167]
Hi, funny behaviour, cosmetic but odd

Go to the Home page and search for ⲱⲥ
You'll get to https://coptic-dictionary.org/entry.cgi?tla=C11263

Then hit the back button on your browser (Chrome 90.0.4430.85 here, Windows 10) and it will think for a split second, and return to the page you are on

The query executed while doing so is

https://coptic-dictionary.org/results.cgi?coptic=%E2%B2%B1%E2%B2%A5&dialect=any&pos=any&definition=&def_search_type=exact+sequence&lang=any

Nothing serious, probably a by-product of a change?

**Comment by amir-zeldes** [comment]
Hi and thanks for reporting - I can't reproduce this on Chrome and Firefox... What browser/OS are you using?

**Comment by m-linssen** [comment]
On mobile here, works fine (Chrome / Android). I'll whip out the laptop and start a fresh session

**Comment by m-linssen** [comment]
Nope, unchanged. Closed the entries browser, opened it, opened new tab - same.
Tried Edge - same

But! If I use incognito mode in either, I have no issues. Odd, and definitely cookie related. But I never use Edge for anything, so that is odd. I deleted all kinds of history there, of all time, but still no change

So: incognito mode Chrome and Edge on Windows work fine, either mode Chrome on Android works fine
